server/athenian/api/internal/reposet.py

```diff
@@ -354,6 +354,26 @@ async def load_prefixer():
                 log.error("account_repos_log does not agree with api_repositories: %s", missing)
             if not repos:
                 raise_no_source_data()
+
+            # add the existing logical repositories as items of the reposet
+            logical_repos = await sdb_conn.fetch_all(
+                select(LogicalRepository.name, LogicalRepository.repository_id).where(
+                    LogicalRepository.account_id == account,
+                ),
+            )
+            for name, physical_repo_id in logical_repos:
+                try:
+                    physical_name = prefixer.repo_node_to_prefixed_name[physical_repo_id]
+                except KeyError:
+                    log.error(
+                        "logical repo %s refers to not existing repo %d", name, physical_repo_id,
+                    )
```

Comment:
Let's accumulate all the missing logicals in a list and log them only once after the cycle.

Reply:
done